src/features/landing/components/Header/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { AppBar, Toolbar } from '@mui/material';
+
+import VilnaMainIcon from '../Icons/VilnaMainIcon/VilnaMainIcon';
```

Comment:
its a hardcoded component written with svg inside, need rework it, we need use only svg icons, not like a component